LottoDiary/LottoDiary/Sources/Present/HomeViewController.swift

```diff
@@ -8,7 +8,7 @@
 import UIKit
 import SnapKit
 
-class ViewController: UIViewController {
+class HomeViewController: UIViewController {
```

Comment:
매서드 디스패치 기억할지 모르겠는데
관련해서 강의나 자료를 다시 찾아보면
class의 경우에 상속하는 경우가 생겨서 dynamic dispatch가 발생하게 되는거 기억날거야 그래서 보통 이런경우(homeviewcontroller는 상속할일이없으니까) 성능 향상을 위해서 static dispatch를 위해서 class 앞에 final 키워드를 붙여주는게 좋아
```suggestion
final class HomeViewController: UIViewController {
```